src/multiplex/server.rs

```diff
@@ -203,11 +210,15 @@ where
                     return Poll::Ready(Err(Error::from_sink_error(e)));
                 }
 
+                #[cfg(feature = "tracing")]
+                tracing::trace!(in_flight = %this.in_flight, pending = %pending.len(), "transport.ready");
```

Comment:
Since `pending` and `in_flight` are `usize`s, they can be recorded as u64s rather than using `fmt::Display`
```suggestion
                tracing::trace!(in_flight = this.in_flight, pending = pending.len(), "transport.ready");
```